clients/aws-sdk-bedrock-runtime/tests/integration/test_bidirectional_streaming.py

```diff
@@ -0,0 +1,305 @@
+    InvokeModelWithBidirectionalStreamOperationInput,
+    InvokeModelWithBidirectionalStreamInput,
+    InvokeModelWithBidirectionalStreamOutput,
+    InvokeModelWithBidirectionalStreamOperationOutput,
+    InvokeModelWithBidirectionalStreamOutputChunk,
+)
+
+
+CHUNK_SIZE = 512
+SILENCE_CHUNKS = 125
+RESPONSE_WAIT_TIME = 3
+
+DEFAULT_SYSTEM_PROMPT = (
+    "You are a friendly assistant. Keep your responses short, "
+    "generally one or two sentences."
+)
+
+START_SESSION_EVENT = """{
+    "event": {
+        "sessionStart": {
+            "inferenceConfiguration": {
+                "maxTokens": 1024,
+                "topP": 0.9,
+                "temperature": 0.7
+            }
+        }
+    }
+}"""
+
+START_PROMPT_EVENT = """{
+    "event": {
+        "promptStart": {
+            "promptName": "%s",
+            "textOutputConfiguration": {
+                "mediaType": "text/plain"
+            },
+            "audioOutputConfiguration": {
+                "mediaType": "audio/lpcm",
+                "sampleRateHertz": 24000,
+                "sampleSizeBits": 16,
+                "channelCount": 1,
+                "voiceId": "matthew",
+                "encoding": "base64",
+                "audioType": "SPEECH"
+            }
+        }
+    }
+}"""
+
+TEXT_CONTENT_START_EVENT = """{
+    "event": {
+        "contentStart": {
+            "promptName": "%s",
+            "contentName": "%s",
+            "type": "TEXT",
+            "interactive": true,
+            "role": "%s",
+            "textInputConfiguration": {
+                "mediaType": "text/plain"
+            }
+        }
+    }
+}"""
+
+TEXT_INPUT_EVENT = """{
+    "event": {
+        "textInput": {
+            "promptName": "%s",
+            "contentName": "%s",
+            "content": "%s"
+        }
+    }
+}"""
+
+CONTENT_END_EVENT = """{
+    "event": {
+        "contentEnd": {
+            "promptName": "%s",
+            "contentName": "%s"
+        }
+    }
+}"""
+
+AUDIO_CONTENT_START_EVENT = """{
+    "event": {
+        "contentStart": {
+            "promptName": "%s",
+            "contentName": "%s",
+            "type": "AUDIO",
+            "interactive": true,
+            "role": "USER",
+            "audioInputConfiguration": {
+                "mediaType": "audio/lpcm",
+                "sampleRateHertz": 16000,
+                "sampleSizeBits": 16,
+                "channelCount": 1,
+                "audioType": "SPEECH",
+                "encoding": "base64"
+            }
+        }
+    }
+}"""
+
+AUDIO_INPUT_EVENT = """{
+    "event": {
+        "audioInput": {
+            "promptName": "%s",
+            "contentName": "%s",
+            "content": "%s"
+        }
+    }
+}"""
+
+PROMPT_END_EVENT = """{
+    "event": {
+        "promptEnd": {
+            "promptName": "%s"
+        }
+    }
+}"""
+
+SESSION_END_EVENT = """{
+    "event": {
+        "sessionEnd": {}
+    }
+}"""
+
+
+@pytest.fixture
+async def bedrock_client():
```

Comment:
Should this be removed? I see we already have a similar fixture added to `conftest.py`.